docs/engineering/backend/readme.mdx

```diff
@@ -0,0 +1,138 @@
+
+    1. The `Organization` is referenced on the `OrganizationAffiliation.organization` attribute.
+    1. The `Location` is referenced on the `OrganizationAffiliation.location` attribute.
+
+- The `Location` child parent relationship is defined by the `Location.partOf` attribute.
+    1. The parent `Location` is referenced on the child's `Location.partOf` attribute.
+
+## Android application
+
+- Update `local.properties` file
+    - Update `FHIR_BASE_URL` value to the `url` of the FHIR Gateway Host
+
+- Data Filtering - configure sync strategy
+    - Update the `application_configuration.json` with the sync strategy for the deployment, e.g. for sync by Location:
+
+    ```json
+    "syncStrategy": ["Location"]
+    ```
+
+> **Note:** Currently the configuration accepts an array but a subsequent update will enforce a single value. See [application_config.json](https://github.com/opensrp/fhircore/blob/main/android/quest/src/main/assets/configs/app/application_config.json)
+
+- Composition JSON
+    - Update the identifier to the value of the application id
+
+    ```json
+    "identifier": {
+        "use": "official",
+        "value": "<app id>"
+    }
+    ```
+
+> **Note:** `identifier.value` above should correspond to `fhir_core_app_id` mentioned in the user management Keycloak section below.
+
+- Update the `sync_config.json` to remove all the non-patient data resources. These should be referenced from the Composition resource so they can be exempted from the Data filter. See [sync_config.json](https://github.com/opensrp/fhircore/blob/b7c24616d4224bd8d16c53b0c2a4f14a1075ce7c/android/quest/src/main/assets/configs/app/sync_config.json)
+
+## FHIR API and configuration resources
+
+1. Deploy the FHIR Store, e.g HAPI
+
+    - The steps here depend on what FHIR Store your are using. To deploy the HAPI FHIR Server using JPA, follow [these](https://github.com/hapifhir/hapi-fhir-jpaserver-starter) steps.
+
+2. `POST` the binary resources referenced in the `composition_config.json`
+
+    - `POST` all non-patient data resources used by the application. Examples of these resources are Questionnaires, StructureMaps, PlanDefinitions, Measures, Libraries, and any Lists referencing other resources.
+
+> **Note:**  When `POST`ing the Binary configuration files in the `composition_config.json` remember to first encode the payload to Base64 before submitting.
+
+> **Note:**  As described in the [FHIR Gateway](backend/info-gateway) section, the server should be in an internal network behind a DMZ and therefore not require authentication, which will be handled by the FHIR Information Gateway.
+
+## Deploy the FHIR Gateway
+
+1. Link to the [Docker image](https://hub.docker.com/r/onaio/fhir-gateway-plugin)
+1. The main documentation for deploying can be found in the [Github READ ME](https://github.com/onaio/fhir-gateway-plugin/blob/main/README.md)For configuration parameters, check out Read Me file for setting environment variables.
```

Comment:
this isn't public, but it also doesn't have docs on deploying, is there a better file to reference here?